modules/runtime/nuxeo-common/src/main/java/org/nuxeo/common/xmap/XValueFactory.java

```diff
@@ -196,6 +197,9 @@ public String serialize(Context context, Object value) {
     public static final XValueFactory CLASS = new XValueFactory() {
         @Override
         public Object deserialize(Context context, String value) {
+            if (StringUtils.isBlank(value)) {
```

Comment:
This change was needed to allow disabling a query maker without redefining the class: see https://jira.nuxeo.com/browse/NXP-30092 for proper handling of such cases, with addition of unit tests.